pkg/policy/api/selector.go

```diff
@@ -56,6 +62,17 @@ func (n EndpointSelector) String() string {
 	return string(j)
 }
 
+// CachedString returns the cached string representation of the LabelSelector
+// within this EndpointSelector.
+func (n *EndpointSelector) CachedString() string {
```

Comment:
How about naming this just as "String()"?

Reply:
There is already a `String()` function which I didn't want to change.